spring-integration-core/src/main/java/org/springframework/integration/store/SimpleMessageStore.java

```diff
@@ -58,6 +58,12 @@
 	private volatile boolean isUsed;
 
 	private volatile boolean copyOnGet = false;
+	
+	private final int maxWaitInMilliseconds;
+	
+	private final static int DEFAULT_MAX_WAIT_IN_MILLISECONDS = 0;
+
+	private final static int DEFAULT_CAPACITY = 0;
```

Comment:
I don't like having multiple hard coded 0s in the code that mean different things. In fact there are 3 places the 0s are, do you know which is which if you inline the code? ;)